Enhance descriptions of the Split Region section (#5750)
pingcap/docs: sql-statements/sql-statement-split-region.md

# Split Region

> For each new table created in TiDB, one [Region](/tidb-storage.md#region) is segmented by default to store the data of this table. This default behavior is controlled by `split-table` in the configuration file. When the data in this Region exceeds the default Region size limit, the Region starts to split into two.

Reviewer: ```suggestion
For each new table created in TiDB, one [Region](/tidb-storage.md#region) is segmented by default to store the data of this table. This default behavior is controlled by `split-table` in the TiDB configuration file. When the data in this Region exceeds the default Region size limit, the Region starts to split into two.
```

There are two types of Split Region syntax:

- The syntax of even split:

{{< copyable "sql" >}}

> ```sql

Reviewer 2: Please add 4 indentations.

`BETWEEN lower_value AND upper_value REGIONS region_num` defines the upper boundary, the lower boundary, and the Region amount. Then the current region will be evenly spilt into the number of regions (as specified in `region_num`) between the upper and lower boundaries.

- The syntax of uneven split:

> {{< copyable "sql" >}}

Reviewer 2: 4 indentations are needed.

    +-----------+-----------+---------+-----------+-----------------+------------------+------------+---------------+------------+----------------------+------------------+
    ```

> 2. Use the `SPLIT` syntax to split a Region for each partition. If you want to divide the data in the `[0,10000]` range of each partition into four Regions, the example statement is as follows:

Reviewer 2: ```suggestion
2. Use the `SPLIT` syntax to split a Region for each partition. Suppose that you want to split the data in the `[0,10000]` range of each partition into four Regions. The example statement is as follows:
```

    {{< copyable "sql" >}}

    ```sql
    split partition table t between (0) and (10000) regions 4;
    ```

>     In the above statement, `0` and `10000` respectively represent the `row_id` of the upper and lower boundaries corresponding to the hotspot data you want to break up.

Reviewer 2: ```suggestion
    In the above statement, `0` and `10000` respectively represent the `row_id` of the upper and lower boundaries corresponding to the hotspot data you want to scatter.
```

    > **Note:**
    >

>     > This example only applies to scenarios where data hotspots are evenly distributed. If the hotspot data is unevenly distributed in the data range you specify, refer to the syntax of uneven split in [Split Regions for partitioned tables](#split-regions-for-partitioned-tables).

Reviewer 2: ```suggestion
    > This example only applies to scenarios where hotspot data is evenly distributed. If the hotspot data is unevenly distributed in a specified data range, refer to the syntax of uneven split in [Split Regions for partitioned tables](#split-regions-for-partitioned-tables).
```

    +-----------+---------------+---------------+-----------+-----------------+------------------+------------+---------------+------------+----------------------+------------------+
    ```

4. You can also split Regions for the index of each partition. For example, you can split the `[1000,10000]` range of the `idx` index into two Regions. The example statement is as follows:

    {{< copyable "sql" >}}

    ```sql
    split partition table t index idx between (1000) and (10000) regions 2;
    ```

#### Examples of Split Region for a single partition

> You can specify the partition to be split. If you want to create a Range table divided into three partitions, the example statement is as follows:

Reviewer 2: This sentence should be in **Line 348**: Suppose that you want to create a Range partitioned table split into three partitions. The example statement is as follows:

        partition p3 values less than (MAXVALUE) );
    ```

> 2. If you want to pre-split the data in the range [0,10000] of the `p1` partition into two Regions, the example statement is as follows:

Reviewer 2: ```suggestion
2. Suppose that you want to split the data in the `[0,10000]` range of the `p1` partition into two Regions. The example statement is as follows:
```

    {{< copyable "sql" >}}

    ```sql
    split partition table t partition (p1) between (0) and (10000) regions 2;
    ```

> 3. If you want to pre-split the data in the range [10000,20000] of the `p2` partition into two Regions, the example statement is as follows:

Reviewer 2: ```suggestion
3. Suppose that you want to split the data in the `[10000,20000]` range of the `p2` partition into two Regions. The example statement is as follows:
```

    +-----------+----------------+----------------+-----------+-----------------+------------------+------------+---------------+------------+----------------------+------------------+
    ```

> 5. If you want to pre-split the [0,20000] range of the index `idx` of the `p1` and `p2` partitions into two Regions, the example statement is as follows:

Reviewer 2: ```suggestion
5. Suppose that you want to split the `[0,20000]` range of the `idx` index of the `p1` and `p2` partitions into two Regions. The example statement is as follows:
```
